frontend/src/components/SearchModal.tsx

```
@@ -0,0 +1,42 @@
+type SearchModalProps = { showModal: boolean; onClose: () => void }
+
+const SearchModal = ({ showModal, onClose }: SearchModalProps) => {
+  if (!showModal) return null
+
+  return (
+    <div className="fixed inset-0 bg-black bg-opacity-50 backdrop-blur-sm flex items-center justify-center z-50">
+      <div className="h-96 w-full flex flex-col justify-between gap-2 bg-[#111318] border border-zinc-800 rounded-xl p-6 max-w-md mx-4">
+        <div>
+          <div className="flex justify-between items-center mb-4">
+            <h2 className="text-white text-lg font-semibold">Search Contacts</h2>
+            <button onClick={onClose} className="text-gray-400 hover:text-white">
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)